src/modules/settings/components/Window.jsx

```diff
@@ -28,12 +28,15 @@ function Page(props) {
 }
 
 function isWindowSmallStandaloneChat() {
-  return window.innerWidth < 600 && window.location.pathname.endsWith('/chat');
+  return (
+    (window.innerWidth < 600 && window.location.pathname.endsWith('/chat')) ||
+    window.location.pathname.endsWith('/live_chat')
+  );
 }
 
 function Window({setHandleOpen}) {
   const [page, setPage] = useState(PageTypes.CHAT_SETTINGS);
-  const [open, setOpen] = useState(false);
+  const [open, setOpen] = useState(true);
```

Comment:
is this really what we want? in general a modal component should be loaded but not open until its prop changes to open, no?

Reply:
for testing*